.github/workflows/CICD.yml

```diff
@@ -151,6 +145,41 @@ jobs:
         use-cross: ${{ matrix.job.use-cross }}
         command: build
         args: --release --target=${{ matrix.job.target }} ${{ matrix.job.cargo-options }}
+    - name: Strip release bin
+      id: strip
+      shell: bash
+      run: |
+        # Figure out suffix of binary
+        EXE_suffix=""
+        case ${{ matrix.job.target }} in
+          *-pc-windows-*) EXE_suffix=".exe" ;;
+        esac;
+
+        # Figure out what strip tool to use if any
+        STRIP="strip"
+        case ${{ matrix.job.target }} in
+          arm-unknown-linux-gnueabihf) STRIP="arm-linux-gnueabihf-strip" ;;
+          aarch64-unknown-linux-gnu) STRIP="aarch64-linux-gnu-strip" ;;
+          *-pc-windows-msvc) STRIP="" ;;
+        esac;
+
+        # Setup paths
+        BIN_DIR="_cicd-intermediates/stripped-release-bin/"
```

Comment:
Any reason why this has a complicated path like this? Do `_*` folders have a special meaning?

Reply:
The main reason for the _ prefix is to follow the established naming convention is this file. Personally I kind of like the _ prefix, because it signals "temporary, short lived dir". If you annoys you, we can of course stop using that prefix :)

The reason for the /stripped-release-bin/ sub dir is for a long term plan of collecting everything CI-temporary under the same dir, so the sub dir is only for "isolation".

Reply:
Makes sense, thanks.